sdk/schemaregistry/azure-schemaregistry-avroencoder/samples/async_samples/encode_and_decode_with_message_content_async.py

```diff
@@ -23,6 +23,29 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
+"""
+FILE: encode_and_decode_with_message_content_async.py
+DESCRIPTION:
+    This sample demonstrates the following:
+     - Authenticating an async SchemaRegistryClient to be used by the AvroEncoder.
+     - Passing in content and schema to the AvroEncoder, to be returned a dict containing
+      encoded content and corresponding content type.
+     - Passing in a dict containing Avro-encoded content and corresponding content type to
```

Comment:
corresponding AvroEncoder content type?

Reply:
Same comment as above. I'm not sure that `AvroEncoder` content type is exactly correct. Maybe more like `Avro content type`? Though that might be misleading b/c it also includes schema ID. But ` content type with schema ID + Avro mime type` seems like a lot. Thoughts?

Reply:
Leave as is then - I think it would be the less confusing option